packages/api/e2e/amino.spec.ts

```diff
@@ -95,7 +95,7 @@ xdescribe('RegenApi with tendermint connection - Amino Tests', () => {
 
       const TEST_MSG_SEND = MsgSend.fromPartial({
         sender: TEST_ADDRESS,
-        recipient: TEST_ADDRESS,
+        recipient: TEST_BUYER_ADDRESS,
```

Comment:
this is because regen-ledger v5.0 doesn't allow sending credits to oneself https://github.com/regen-network/regen-ledger/pull/1674